doc/rst/source/supplements/geodesy/psvelo.rst

```diff
@@ -22,7 +22,7 @@ Synopsis
 [ |-K| ]
 [ |-L| ]
 [ |-N| ] [ |-O| ] [ |-P| ]
-[ |-S|\ *symbol*/*scale*\ [/*args* ] ]
+[ |-S|\ *symbol*/*args*\ [**+f**\ *font*] ]
```

Comment:
No slash after *symbol*.

Reply:
OK fixed via update the same way as we do in meca.